data/101/939/383/101939383.geojson

```diff
@@ -2,23 +2,24 @@
   "id": 101939383,
   "type": "Feature",
   "properties": {
+    "au-psma:modified":"2019-08-20",
     "edtf:cessation":"uuuu",
     "edtf:inception":"uuuu",
-    "geom:area":0.000155,
-    "geom:area_square_m":1693736.579675,
-    "geom:bbox":"152.99540094,-27.9941850045,153.024792128,-27.9590956113",
-    "geom:latitude":-27.982289,
-    "geom:longitude":153.010439,
+    "geom:area":0.003881,
+    "geom:area_square_m":42368958.950133,
+    "geom:bbox":"153.074295,-28.037276,153.142768,-27.953549",
+    "geom:latitude":-27.999603,
+    "geom:longitude":153.111316,
     "iso:country":"AU",
-    "lbl:latitude":-27.989464,
-    "lbl:longitude":153.013778,
-    "lbl:max_zoom":18.0,
+    "lbl:latitude":-27.999603,
+    "lbl:longitude":153.111316,
+    "lbl:max_zoom":18,
     "lbl:min_zoom":13.5,
     "mps:latitude":-27.989464,
     "mps:longitude":153.013778,
     "mz:hierarchy_label":1,
     "mz:is_current":1,
-    "mz:min_zoom":12.0,
+    "mz:min_zoom":12,
```

Comment:
Ah this might be caused when I swap the geometry as I need to parse the JSON, make the changes then write it back out again.
How much of a problem is this? I would assume that most tools should handle 12.0 and 12 equally well?